config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'volunteers/index'
```

Comment:
Should you use Resourceful routing? If there are going to be show, update, etc actions you should.

Reply:
Okie dokie!